### DISCUSSION TITLE
Emporia vu with PG&E meter to arduino?

### DISCUSSION THREAD

**Discussion by wannabuy** [emporia-vue-local/esphome#39] [General]
Hi All,
Forgive my noobism, but has someone already crossed this bridge? maybe have a interface for sale? I'm not looking for the vue2 with its many sensors, but a basic panel for reading the utility meter power (both positive and negative) that connected to arduino or some other micro. I would use the micro side to control output ports to turn on or off chargers when +/- limits are exceeded...
Thanks,
Bryan

**Comment by flaviut** [comment]
Are you looking for something like this? https://github.com/klaasnicolaas/home-assistant-glow

I'm not familiar with this subject, but if you buy an Vue 2 with just the mains clamps, I can answer your questions about that.

**Comment by binfill3** [comment]
Perhaps you want this? https://github.com/jrouvier/esphome-emporia-vue-utility It's a 25 dollar purchase and not related to the vue2, apart from being made by the same company. You need to have a recent-ish Zigbee enabled meter - these smart meters have been the standard for PGE since several years now.
